pope/.htaccess

```diff
@@ -0,0 +1,29 @@
+Options +FollowSymLinks
+# Turn off MultiViews
+Options -MultiViews
+
+# Directive to ensure *.rdf files served as appropriate content type,
+# if not present in main apache config
+AddType application/rdf+xml .rdf
+AddType application/rdf+xml .owl
+AddType text/turtle .ttl
+AddType application/n-triples .n3
+
+RewriteEngine on
+
+#Rewrite rules for PoPE vocabulary  
+RewriteCond %{HTTP_ACCEPT} !application/rdf\+xml.*(text/html|application/xhtml\+xml|text/\*|\*/\*)
+RewriteCond %{HTTP_ACCEPT} text/html [OR]
+RewriteCond %{HTTP_ACCEPT} application/xhtml\+xml [OR]
+RewriteCond %{HTTP_ACCEPT} text/\* [OR]
+RewriteCond %{HTTP_ACCEPT} \*/\* [OR]
+RewriteCond %{HTTP_USER_AGENT} ^Mozilla/.*
+RewriteRule ^pope$ https://foodsentimentobservatory.github.io/PoPE/index-en.html  [R=303,NE,L]
```

Comment:
Rules are relative to the current directory.  This will result in `https://w3id.org/pope/pope`.  You probably mean to change these to `^$`.

Reply:
Thnaks